config/default/user.role.contributor.yml

```diff
@@ -67,10 +68,7 @@ permissions:
   - 'create video media'
   - 'delete humsci entity'
   - 'delete own files'
-  - 'delete own image media'
-  - 'delete own video media'
   - 'dropzone upload files'
```

Comment:
Are all these permission changes supposed to be part of this?

Reply:
@codechefmarc @cienvaras

Reply:
No, those should not be there, good catch. I'll remove and resubmit.

Reply:
Fixed.